src/API/Instrumentation/SpanSuppression/SemanticConvention.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Instrumentation\SpanSuppression;
```

Comment:
Move to `OpenTelemetry\API\Trace\SpanSuppression` as it only applies to traces?

Reply:
Yes, I think these now belong in API\Trace